sat/util.py

```diff
@@ -661,12 +662,12 @@ def get_s3_resource():
             )
     except OSError as err:
         LOGGER.error(f'Unable to load configuration: {err}')
-        raise SystemExit(1)
+        raise S3ResourceCreationError(str(err))
```

Comment:
I find that error and exception handling is a delicate art. Generally, I think if a function is going to raise an exception to the caller when a problem is encountered, it is not usually necessary to also log an error message. The caller may decide that this error condition is non-fatal and may want to log a warning and continue instead. In fact, I think that is the case for `sat showrev`.

If we get rid of the `LOGGER.error`, it would be helpful to add context to the error message passed to the constructor of `S3ResourceCreationError`. It's also possible to keep a reference to the original exception in the new exception. Putting all that together, I'd suggest something like this:

```suggestion
        raise S3ResourceCreationError(f'Unable to load S3 key: {err}') from err
```

Here's an example in my interpreter of the logged errors before my change:

```
>>> try:
...   s3 = get_s3_resource()
... except Exception as err:
...   print(err)
...
Unable to load configuration: [Errno 2] No such file or directory: '/Users/haasken/.config/sat/my_s3_access_key'
[Errno 2] No such file or directory: '/Users/haasken/.config/sat/my_s3_access_key'
```

And after my suggested change:

```
>>> from sat.util import get_s3_resource
>>> try:
...   s3 = get_s3_resource()
... except Exception as err:
...   print(err)
...
Unable to load S3 key: [Errno 2] No such file or directory: '/Users/haasken/.config/sat/my_s3_access_key'
```